spec/unit/work_spec.rb

```diff
@@ -207,182 +219,186 @@ def run
   end
 
   describe "when encountering an error" do
-    it "should exponentially back off the job" do
-      ErrorJob.enqueue
+    describe "when using the default strategy" do
```

Comment:
There's no context for the case where a different strategy is being used. Feels like the delegation is worth testing inside Que (though not the fine-grained behaviour of what we're doing in `Que::Retry`).